### ISSUE TITLE
[Question] skipLibCheck for 30 libraries

### ISSUE THREAD

**Issue by Wockeez** [openlayers/openlayers#14814]
I looked at issue #14496, I have about the same situation, BUT, I have openlayers as a angular library dependency (example: @lib/map)

![image](https://github.com/openlayers/openlayers/assets/59292278/522ed027-c48d-4554-88cc-414546fba745)

I have another **30+**  angular libraries that have @lib/map installed, with npm i openlayers is also installed as a dependency

Now any library where I have @lib/map present will not compile

Do I need to put skipLibCheck: true in all my 30+ libraries?

**Comment by jahow** [comment]
Thanks for sharing! Your solution looks quite involved (creating a fork and publishing a custom version of the library), are you positive there was no simpler way to make the errors go?

**Comment by Wockeez** [comment]
> Thanks for sharing! Your solution looks quite involved (creating a fork and publishing a custom version of the library), are you positive there was no simpler way to make the errors go?

I investigated this problem and came to the conclusion that typescript 5 is incompatible with ts 3.5 (no such errors in d.ts with typescript 5). An analogous solution to this problem is to update the version of angular in my libraries, at least to ts 4. As I said before I have 30+ libraries on angular 8, I think I have found the right solution to this problem, even though it is complicated.
